vllm/utils.py

```diff
@@ -445,19 +451,28 @@ async def iterate_with_cancellation(
     at least once per second to check for client cancellation.
     """
 
+    loop = asyncio.get_running_loop()
+
     # Can use anext() in python >= 3.10
```

Comment:
We should probably move these comments to the `_next_task` function

Reply:
Agree, I noticed that too actually but wasn't sure about triggering another CI run for it lol.. but let me make that update